.env

```diff
@@ -0,0 +1 @@
+PORT=8082
```

Comment:
We don't actually need this file for running the proxy server for local CMS testing, so you can delete it